api/src/main/resources/workbench.yaml

```diff
@@ -66,6 +66,25 @@ paths:
           schema:
             $ref: "#/definitions/BugReport"
 
+  # TODO(dmohs): If the username is not present, this response with 500 Server Error. It should
+  # respond with 400 Bad Request.
+  /api/v1/is-username-taken:
```

Comment:
So is the idea that as the user is typing you're going to call this endpoint?

(If we were just going to check for the username being present at the point that they submit a form, we probably wouldn't need an endpoint like this; it's better to have the directory service do the check in the same transaction as creating the account.)